src/brsTypes/components/RoAssociativeArray.ts

```diff
@@ -93,18 +92,49 @@ export class RoAssociativeArray extends BrsComponent implements BrsValue, BrsIte
         // method with the desired name separately? That last bit would work but it's pretty gross.
         // That'd allow roArrays to have methods with the methods not accessible via `arr["count"]`.
         // Same with RoAssociativeArrays I guess.
-        let indexValue = this.modeCaseSensitive ? index.value : index.value.toLowerCase();
-        return this.elements.get(indexValue) || this.getMethod(index.value) || BrsInvalid.Instance;
+        return (
+            this.findElement(index.value, isCaseSensitiveGet) ||
```

Comment:
I think you can get rid of `findElement` function and just do the following:
```
this.elements.get(this.findElementKey(index.value), isCaseSensitiveGet)
```
because if the key is `undefined` the map should return that as well

Reply:
![image](https://user-images.githubusercontent.com/33808928/114775791-60c49900-9d7a-11eb-9764-7c36014f6390.png)
looks like no, it can not accept `undefined`

Reply:
![image](https://user-images.githubusercontent.com/33808928/114775967-98334580-9d7a-11eb-8c0c-ba57324681bc.png)